README.md

```diff
@@ -55,6 +55,29 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Files
+
+### `cat` 
+
+> Streams the data contained by an IPFS object(s) at the given IPFS multihash..
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.cat(multihash, [callback])
+
+`multihash` is a [multihash][] which can be passed as
+
+- Buffer, the raw Buffer of the multihash (or of and encoded version)
+- String, the toString version of the multihash (or of an encoded version)
```

Comment:
All Qm strings are currently base58 encoded but they don't have to be. Think the idea is to be able to support more encodings for string represented multihashes in the future. Since base58 encoding the multihash binary to a string is I assume for human readably purposes and if a better binary -> string encoder comes around then others would want it represented that way.